content/container_insights/setupalarm.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup CloudWatch Alarm"
+chapter: false
+weight: 3
+---
+#### Setup CloudWatch Alarm
+
+In this section, we will setup a CloudWatch alarm and send notification using SNS the number of Pod restarts goes greater than 1 in a 5 minute period.
+
+##### Steps
+
+* Navigate to [CloudWatch Alarms](https://console.aws.amazon.com/cloudwatch/home#alarmsV2:)
+* Click on **Create alarm**
+* In the **Specify metric conditions screen** click **Select metric**
+* Select **Container Insights** and then **ClusterName, Namespace, PodName**.
+* Then check the **pod_number_of_container_restarts** metric as shown below
+
+![Container Insights](/images/ContainerInsights7.png)
+
+* Click **Select metric**
+* In the **Specify metric and conditions** screen, change the statistic to **Sum**
+* Enter **1** in the **Define the threshold value** textbox. Click **Next**
+* In the **Configure actions** screen, ensure **in Alarm** is selected and select **Select an existing SNS topic** if you have an SNS topic already in your account you want to use for this purpose. If not, select **Create new topic**
+
+![Container Insights](/images/ContainerInsights8.png)
+
+* Click on **Create topic** and then click **Next**
+* In the **Add a description** screen, enter the alarm name as **Pod Restart Count** and click **Next**
+
+![Container Insights](/images/ContainerInsights10.png)
+
+* Review the details in the following screen and click **Create alarm**
+
+##### Trigger the alarm
+
+Get the name of the **ecsdemo-frontend** pod using the following command
+
+```
+kubectl get pods
```

Comment:
I'd just add both command and expected output. And Remove the table below, very nit picky but it helps with consistency
For ex, 
$ kubectl get pods --all-namespaces
NAMESPACE        NAME                                              READY   STATUS    RESTARTS   AGE
kube-system      aws-node-5pzjk                                    1/1     Running   0          3d22h
kube-system      aws-node-7dbgq                                    1/1     Running   0          3d22h
kube-system      aws-node-fwvd5                                    1/1     Running   0          3d22h